### DISCUSSION TITLE
Create group construct for hosts

### DISCUSSION THREAD

**Discussion by rkelleyrtp** [turtle0x1/LxdMosaic#520] [Ideas]
**Is your feature request related to a problem? Please describe.**
When adding servers to LXDMosaic, it would be very helpful/useful to create a group that contains the hosts.  In my environment, we have three "sets" of LXD Servers: Admin Servers, QA Servers, and Production Servers.  Currently, each server is added as an individual server in the Standalone Hosts list.  It would be very helpful to create a Group for each server type.  Then, the servers will be placed in the appropriate group (maybe via tag?)

Today:
Standalone Hosts
 --> Admin Server 1
 --> Admin Server 2
 --> Admin Server 3
 --> QA Server 1
 --> QA Server 2
 --> QA Server 3
 --> Prod Server 1
 --> Prod Server 2
 --> Prod Server 3

In a grouping format:

Standalone Hosts

 **Group: Admin**
 --> Admin Server 1
 --> Admin Server 2
 --> Admin Server 3

 **Group: QA**
 --> QA Server 1
 --> QA Server 2
 --> QA Server 3

**Group: Prod**
 --> Prod Server 1
 --> Prod Server 2
 --> Prod Server 3

By default, each standalone server could be in it's own group.

**Comment by turtle0x1** [comment]
I see the need for this but its more complicated than it sounds (and introduces another level of loops).

Its not a no but I cant slide this into the next release :smile:

**Comment by turtle0x1** [comment]
Migrating to "discussions" because its not a "bug fix" and is more "big picture"

Thanks
turtle0x1
